itest/addrs_test.go

```diff
@@ -35,7 +35,7 @@ func testAddresses(t *harnessTest) {
 	// assets made above.
 	secondTarod := setupTarodHarness(
 		t.t, t, t.lndHarness.BackendCfg, t.lndHarness.Bob,
-		t.universeServer, false,
+		t.universeServer, false, false,
```

Comment:
Hmm, could we turn both of these two new boolean into functional options so we don't need to add them where they have the default values?